.github/workflows/maven.yml

```
@@ -32,14 +32,15 @@ jobs:
 
     strategy:
       matrix:
-        java-version: [8, 11]
+        java-version: [8, 11, 17, 21]
 
     steps:
     - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)